sdk/core/azure-core/azure/core/pipeline/transport/requests_basic.py

```diff
@@ -181,11 +183,12 @@ class RequestsTransport(HttpTransport):
 
     _protocols = ['http://', 'https://']
 
-    def __init__(self, configuration=None, session=None, session_owner=True):
+    def __init__(self, session=None, session_owner=True, **kwargs):
```

Comment:
I would actually move `session` and `session_owner` to `**kwargs`. I don't expect them to be used in the majority of cases.